packages/flutter/lib/src/widgets/autocomplete.dart

```diff
@@ -514,9 +528,13 @@ class _RawAutocompleteState<T extends Object> extends State<RawAutocomplete<T>>
             shortcuts: _shortcuts,
             child: Actions(
               actions: _actionMap,
-              child: CompositedTransformTarget(
-                link: _optionsLayerLink,
-                child: fieldView,
+              child: Visibility(
```

Comment:
This `Visibility` widget can be avoided, by switching between `fieldView` or `CompositedTransformTarget(child: fieldView)` directly, based on the presence of the layer link.